tests/ci/run_fips_tests.sh

```diff
@@ -10,20 +10,17 @@ fips_build_and_test -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1
 # The AL2 version of Clang does not have all of the required artifacts for address sanitizer, see P45594051
 if [[ "${AWSLC_NO_ASM_FIPS}" == "1" ]]; then
   if [[ ("$(uname -p)" == 'x86_64'*) ]]; then
-    echo "Building with Clang and testing AWS-LC in FIPS RelWithDebInfo mode with address sanitizer."
-    fips_build_and_test -DASAN=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_SHARED_LIBS=1
+    echo "Building with Clang and testing AWS-LC in FIPS Release mode with address sanitizer."
+    fips_build_and_test -DASAN=1 -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1
   else
     # See the comment in run_posix_santizers.sh for more context. ASAN on Arm has a huge performance impact on ssl_test
     # which causes it to take over 2 hours to complete.
-    echo "Building with Clang and testing AWS-LC in FIPS RelWithDebInfo mode with address sanitizer only running crypto_test"
-    run_build -DASAN=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_SHARED_LIBS=1
+    echo "Building with Clang and testing AWS-LC in FIPS Release mode with address sanitizer only running crypto_test"
+    run_build -DASAN=1 -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1
```

Comment:
Is this the ASAN build we had a discussion about? Is it running on Aarch64?

Reply:
> Is this the ASAN build we had a discussion about? 

Yes.

> Is it running on Aarch64?

Yes.